lib/solidus_importer/processors/product_images.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module SolidusImporter
+  module Processors
+    class ProductImages < Base
+      def call(context)
+        @data = context[:data]
+        @entity = context[:entity]
+        context.merge!(check_data || save_images)
+      end
+
+      private
+
+      def check_data
+        if @data.blank?
+          { success: false, messages: 'Missing input data' }
+        elsif @data['Image Src'].blank?
+          {}
+        elsif !@entity || !@entity.is_a?(Spree::Product) || !@entity.valid?
+          { success: false, messages: 'Target entity must be a valid product' }
+        end
+      end
+
+      def prepare_image
+        Spree::Image.new(
+          attachment: File.open(@data['Image Src']),
```

Comment:
Are we positive that the `Image Src` is a file path? I would have expected it to be a url instead…

Reply:
You are right 😅
What do you think it's better? To assume that we will always have remote URLs? To look for HTTP urls otherwise the resource is local?

Reply:
Probably safe to always expect URLs and leave local files for customizations 👍